readme.txt

```diff
@@ -0,0 +1,71 @@
+=== wp-cldr ===
```

Comment:
Is a `readme.txt` required? It'll be a pain to keep this in sync w/ the md file.